zhaquirks/thirdreality/plug.py

```diff
@@ -0,0 +1,39 @@
+"""Third Reality Plug devices."""
+
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class ThirdRealityPlugCluster(CustomCluster):
+    """Third Reality's plug private cluster."""
+
+    cluster_id = 0xFF03
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Define the attributes of a private cluster."""
+
+        reset_summation_delivered: Final = ZCLAttributeDef(
+            id=0x0000,
+            type=t.uint8_t,
+            is_manufacturer_specific=True,
+            name="Reset_summation_delivered",

```

Comment:
The `name` is already defined by the attribute name when using this definition style, we should remove it.
```suggestion
```